voice/voicegateway/gateway.go

```diff
@@ -131,7 +124,7 @@ func (c *Gateway) start(ctx context.Context) error {
 
 		// Close can be called with the mutex still acquired here, as the
 		// pacemaker hasn't started yet.
-		if err := c.Close(); err != nil {
+		if err = c.Close(); err != nil {
 			wsutil.WSDebug("VoiceGateway: Failed to close after start fail: ", err)
 		}
```

Comment:
Changing this would overwrite `err` to `nil`, if `c.Close` succeeds, which is undesired.

```suggestion
		if closeErr := c.Close(); closeErr != nil {
			wsutil.WSDebug("VoiceGateway: Failed to close after start fail: ", closeErr)
		}
```